Source/santad/SNTExecutionController.mm

```diff
@@ -339,8 +339,7 @@ - (void)validateExecEvent:(const Message &)esMsg postAction:(bool (^)(SNTAction)
   bool holdAndAsk = false;
```

Comment:
nit: Now that this is stored in the SNTCachedDecision, let's ditch this local variable that is now duplicative.